src/utils/tokenUtils.js

```diff
@@ -12,8 +12,17 @@ const storeToken = response => {
   return Promise.reject(response.statusText);
 };
 
+const storeLinkedInToken = response => {
```

Comment:
We don't need to keep `storeToken` function anymore, I guess we can just delete `storeToken` function and rename this to `storeToken`